os/arch/arm/src/amebad/amebad_signature.c

```diff
@@ -20,17 +20,18 @@
  * Included Files
  ****************************************************************************/
 #include <tinyara/config.h>
+#include <tinyara/signature.h>
 #include <sys/types.h>
```

Comment:
Because we change OK to SIG....VALID, this is unnecessary. Let's remove this including.

Reply:
Erased